testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/cdi/injection/InjectionTest.java

```diff
@@ -74,7 +75,8 @@
 @RunWith(Arquillian.class)
 @RunAsClient
 @Category({
-    ExpectedFailingWithStandaloneMicroprofileConfiguration.class // MP is missing jboss.naming.context.java.jboss.exported.jms.RemoteConnectionFactory
+    ExpectedFailingWithStandaloneMicroprofileConfiguration.class, // MP is missing jboss.naming.context.java.jboss.exported.jms.RemoteConnectionFactory
+    NotForBootableJar.class // needs ejb + standalone-full
```

Comment:
Is it possible to use standalone-full for bootable-jar testing here?

Reply:
I don't think so. The supported layers combinations never bring up a standalone-full. I discussed this with @jfdenise who could add more input, in case it is needed.

Reply:
Right, sorry, standalone profiles probably doesn't make sense in this context. So what layer is missing for correct execution of this test (from standalone-full)? Messaging?

Reply:
IIRC was messaging but there were several more things missing. In fact I tried and added the new messaging layer but it didin't work.

Reply:
It would be great to describe more details about reason for ignoring this testcase to java comment ...

Reply:
Ok, adding more detail about the fact that a 250 MB bootable JAR is basically useless.

Reply:
> which is useless

Hmm, ok, but what is missing in bootable jar to have this working?

Reply:
Messaging was missing for sure, and I added it. Then it failed again, and I discussed the thing with the feature developer, who made his experiments as well and he got to the conclusion that a full server was needed to make this work.
Maybe @jfdenise can elaborate a bit if what you need to know is the full list of subsystems that should in place.

Reply:
I don't think that this blocks this PR. But I'm wondering whether this is some issue that could affects users. @jfdenise can you please provide more details on this?

Reply:
I don't have the list of needed subsystems, I realized that a standalone-full.xml was used in the test (embedded broker for example). I made the bootable JAR to provision a full config and the test passed. BTW, I also noticed that I had to remove one of the 2 executions in the arquilian-bootable.xml. We are starting 2 server that unzip there content in the same location (install-dir). This breaks messaging journal that doesn't seem to like the WildFly directory to be re-written.

Reply:
>  We are starting 2 server that unzip there content in the same location (install-dir)

+1, this looks like a TS bug. @fabiobrz we can for example add some suffix to the second arquillian container

Reply:
Thanks @marekkopecky and @jfdenise, done.